install.php

```diff
@@ -1,3 +1,40 @@
 <?php
-$_GET["mod"] = "install";
-include_once("index.php");
+
+//raise error reporting - could be interesting and this is autmomatically reset when index.php is executed
+error_reporting(E_ALL);
+
+
+/**
+* Check for common failures
+*
+* Small function to check against a number of dependencies that cause install.php to fail without showing any error details
+* @return bool check result. True if error found, false if all OK
+*/
+function checkdeps(){
+
+	$error =false;
+	$errmsg='The following issues were found while preparing to run the installation:<br/>'. PHP_EOL;
```

Comment:
The `$var=value` is inconsistent.
Sometimes `$var=value` or `$var =value` or `$var = value`. Personally i think `$var = value` is more readable. Do you agree?

Reply:
yup ;)